Update index page to display content from the Wordpress API

moudev opened the pull request:
# Antes de enviar el Pull Request
- [x] Confirmo que revisé que mis cambios funcionan
- [x] He adjuntado una imagen de cómo se ven mis cambios (si aplica)
- [x] He ejecutado prettier en los archivos de mis cambios

# Descripción

- **Los pasos para configuración están en la [wiki del repositorio](https://github.com/svconectado/website/wiki/Gu%C3%ADa-para-implementar-de-forma-local-el-website-de-%22El-Salvador-Conectado%22)**

- Actualizando la página principal con la función de mostrar datos extraídos de la API de Wordpress, estos datos se extraen de una página que necesita tener el nombre _**content-page-home**_ porque la página será buscada por su URL Slug en el archivo de la página principal. 

- Fracción de código con la consulta hecha a Wordpress

  ```javascript
      wpgraphql {
          page(id: "content-page-home", idType: URI) {}
      }
  ```

- Actualización del componente de encabezado para mostrar datos de la API de Wordpress, los datos son pasados desde la página principal que es el lugar donde se hace la extracción de información, luego pasada a el Layout y posteriormente al component Header.

- Agregando en la página principal las secciones con información extraída de la API de Wordpress;
  - Nuestras motivaciones
  - Sobre nosotros
  - Nuestras iniciativas

- Si la página _**content-page-home**_ no se ha creado, se mostrará un mensaje con las instrucciones. 

- Agregando validaciones en el archivo gatsby-node para crear páginas a partir de datos de la API de Wordpress

- Actualización del archivo .eslintrc con la regla para saltar el error que ocurrió por agregar las validaciones en gatsby-config

- Actualización de la wiki del repositorio con las instrucciones de configuración. [Guía para implementar de forma local el website de "El Salvador Conectado"](https://github.com/svconectado/website/wiki/Gu%C3%ADa-para-implementar-de-forma-local-el-website-de-%22El-Salvador-Conectado%22)

# Capturas

- **[Mobile] Página principal**

![Screenshot_2020-04-20 Home(2)](https://user-images.githubusercontent.com/13499566/79806057-c4151a00-8324-11ea-8758-0e9903c423b4.png)

- **[Tablet] Página principal**

![Screenshot_2020-04-20 Home(1)](https://user-images.githubusercontent.com/13499566/79806068-caa39180-8324-11ea-98c4-26847859a421.png)

- **[Desktop] Página principal**

![Screenshot_2020-04-20 Home](https://user-images.githubusercontent.com/13499566/79806076-cecfaf00-8324-11ea-84c2-89c7f914ce50.png)

- **[Mobile] Mensaje cuando no existe página creada**

![Screenshot_2020-04-20 Home(4)](https://user-images.githubusercontent.com/13499566/79806086-d42cf980-8324-11ea-8276-67d18bd4a8f8.png)

- **[Desktop] Mensaje cuando no existe página creada**

![Screenshot_2020-04-20 Home(3)](https://user-images.githubusercontent.com/13499566/79806096-d727ea00-8324-11ea-92cc-2d0a22a27ebf.png)

moudev committed:
Update index page to get content from Wordpress API

Updating index page with the feature to show data extracted
from the Wordpress API, this data is extracted from
a page that needs to have the name 'content-page-home' because
the page will be searched by its URL Slug.

If the page hasn't been created a message with the instructions
will be displayed. The index page gets and passes it to the
layout component that contains the Header component, after
the data is passed from layout component to Header component.

Important; for this to works it is important install the
Advanced Custom Fields plugin in our WP backend and
create specific fields, this fields names are in the index
page graqhql query.
https://www.wpgraphql.com/acf/

  - Adding validations in gatsby-node file to create pages
    from data of the Wordpress API
  - Updating .eslintrc file whit rule to pass error that
    occurred for the validations added in gatsby-config
  - Updating the index page to get custom fields from the Wordpress
    API
  - Updating the header component to display data from the
    Wordpress API

eduarmreyes commented on src/components/header.js:
```diff
@@ -45,24 +50,39 @@ const Header = () => (
+                        </span>
+                        <pre>
+                          {`
+<h1 class="message">
```
esto lo podés poner en una variable para evitar romper el formato

eduarmreyes reviewed (changes requested):
revisa mi comentario

moudev commented:
Se cambió a draft para usar;
- https://www.advancedcustomfields.com/
- https://www.wpgraphql.com/acf/

Para almacenar información dentro de campos especificos, haciendo consulta con graphql para obtenerlos y solo asignarlos como variables dentro del código, por tal motivo hay que revertir los cambios de clases en el header component ya que ahora solo se extraerán variables para ser mostradas.

moudev committed:
Update index page to show data from Wordpress API

Adding styled sections of information in the index page,
this data is from the WP API. The sections added are;

- Our motivations
- About Us
- Our Initiatives

  - Adding text-color in tailwind config file
  - Adding sections in the index page file, the data
    of these sections are custom fields created in
    the WP backend with the plugin "Advanced Custom Fields",
    if you want to see the names of these ACFs you can look at
    'src/pages/index.js' file on its graphql query or
    you can look at;
    https://github.com/svconectado/website/wiki

moudev committed:
Add author credits of images used in header and the index page

moudev committed:
Update gatsby-config file to use the .env file

  - Adding dotenv package

moudev committed:
Add prettier plugin in ESLint rules

  - Adding prettier rules in the ESLint config file
  - Fixing warnings files with the rules of prettier

moudev committed:
Update blog and page templates, one file for each type

  - Adding page-template with the difference that this template
    doesn't show the title and author information
